test/integration/short/prepare-simulator-tests.js

```diff
@@ -0,0 +1,166 @@
+'use strict';
+var assert = require('assert');
+var util = require('util');
+
+var Client = require('../../../lib/client');
+var types = require('../../../lib/types/index');
+var utils = require('../../../lib/utils');
+var helper = require('../../test-helper');
+var reconnection = require('../../../lib/policies/reconnection');
+var simulacron = require('../simulacron');
+
+describe('Client #prepare #simulacron', function () {
+  this.timeout(20000);
+  describe('Preparing statements on nodes behavior', function () {
+    var nodes = "5";
+    var clusterName = "simulacronTest";
+    var cassandraVersion = "3.10";
+    var sCluster = null;
+    var client = null;
+    var query = util.format('SELECT * FROM ks.table1 WHERE id1 = ?');
+    before(function (done) {
+      simulacron.start(done);
+    });
+    beforeEach(function (done) {
+      utils.series(
+        [
+          function startCluster(next) {
+            sCluster = new simulacron.SimulacronCluster();
+            sCluster.start(nodes, cassandraVersion, "", clusterName, true, 1, next);
```

Comment:
Except the amount of nodes and the callback, the rest of the parameters can be optional with a default.
You could use an options parameter or something like that:

```js
sCluster.start(nodes, { optionalParams: value... }, next);
```

About the cassandra version, you can use `helper.getCassandraVersion()` internally.